spinalcordtoolbox/resample/nipy_resample.py

```diff
@@ -0,0 +1,176 @@
+#########################################################################################
+#
+# Resample data.
+#
+# ---------------------------------------------------------------------------------------
+# Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Sara Dupont
+# Modified: 2017-11-22 (moved the code to spinalcordtoolbox module)
+#
+#
+# About the license: see the file LICENSE.TXT
+#########################################################################################
+import sys
+import commands
+from copy import deepcopy
+
+import sct_utils as sct
+
+from msct_image import Image
+from msct_parser import Parser
+
+import nipy
+import numpy as np
+from nipy.algorithms.registration.resample import resample as n_resample
+
+def resample_image(input_image, new_size, new_size_type,
+                   interpolation='linear', verbose=1):
+    """This function will resample the specified input
+    image to the target size.
+
+    :param input_image: The input image.
+    :param new_size: The target size, i.e. 0.25x0.25
```

Comment:
If `0.25x0.25` is a string, put in quotes to be clearer wrt/ param type

Reply:
Added quotes.